app/views/unpublishes/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class='well'>
+  <%= form_for @unpublish, url: guide_unpublishes_path(@guide) do |f| %>
+    <h2>Unpublish <small><%= @guide.slug %></small></h2>
+
+    <div class="form-group">
+      <%= f.label :new_path do %>
+        Redirect to (<span class="text-info">only published guides and topics will appear in this list</span>)
+      <% end %>
+      <%=
+        f.select :new_path,
+          grouped_options_for_select(@select_options),
+          {include_blank: "Please choose a path to redirect to"},
+          {class: "select2 slug-migration-redirect-to input-md-12 form-control"}
```

Comment:
I can't see what the `slug-migration-redirect-to` class does here or in the original slug migrations actually 😕  A left over from the past?